hedera-node/hedera-evm-api/src/main/java/com/hedera/services/evm/store/contracts/HederaEvmWorldStateTokenAccount.java

```diff
@@ -26,16 +26,17 @@
 import org.hyperledger.besu.evm.account.Account;
 import org.hyperledger.besu.evm.account.AccountStorageEntry;
 
-public class WorldStateTokenAccount implements Account {
-    public static final String TOKEN_BYTECODE_PATTERN = "fefefefefefefefefefefefefefefefefefefefe";
-    public static final String TOKEN_CALL_REDIRECT_CONTRACT_BINARY =
+public class HederaEvmWorldStateTokenAccount implements Account {
+    private static final String TOKEN_BYTECODE_PATTERN = "fefefefefefefefefefefefefefefefefefefefe";
+    private static final String REDIRECT_CONTRACT_BINARY =
```

Comment:
This bytecode is still HTS specific (`610167` is `PUSH2 0x167`) so the name should still reflect that.

Reply:
Renaming the field back to _TOKEN_CALL_REDIRECT_CONTRACT_BINARY_ cause a sonar "token exposure issue". @SuppressWarnings added to handle this.